sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -501,6 +502,7 @@ group varying "Varying linkage"
 		case differing_interpolation_2
 			version 300 es
 			desc "varying interpolation different (smooth vs. centroid)"
+			expect link_fail
```

Comment:
C++ dEQP passed without link_fail:
https://android.googlesource.com/platform/external/deqp/+/deqp-dev/data/gles3/shaders/linkage.test#497

But according GLSL spec "Section 4.3.9 Interpolation", "The type and presence of the interpolation qualifiers and storage qualifiers and invariant qualifiers of variables with the same name declared in all linked shaders must match, otherwise the link command will fail."

So marked as link_fail here.  The link_fail error is reported in command buffer before it was sent to driver.  So it behaviors differently than C++.